### ISSUE TITLE
Adding multiple tags to a cloudHSM cluster does not work

### ISSUE THREAD

**Issue by mudlark** [terraform-providers/terraform-provider-aws#9871]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

$ terraform -v
Terraform v0.12.6
+ provider.aws v2.24.0

### Affected Resource(s)

* aws_cloudhsm_v2_cluster

### Debug Output

https://gist.github.com/mudlark/0a917aa4863023da1b5843d23ba63d1e

### Expected Behavior

4 tags created:

```    tags.%:                   "4"
    tags.Name:                "test_hsm_cluster"
    tags.ProjectName:         "infra/cloudhsm-test"
    tags.budget:              "infra"
    tags.org:                 "infra"
```

### Actual Behavior

1 tag created (org: infra in this case, but it's random)

### Steps to Reproduce

Create a cloudHSM cluster resource with more than 1 tag.

1. `terraform apply`

### Important Factoids

I've fixed this in PR 9741

### References

* https://github.com/terraform-providers/terraform-provider-aws/pull/9741

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.34.0 of the Terraform AWS Provider, between now and next Thursday.

**Comment by bflad** [comment]
This has been released in [version 2.34.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2340-october-31-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
